### ISSUE TITLE
ranger crashes when initialising git repo with files in it

### ISSUE THREAD

**Issue by Ablesius** [ranger/ranger#1056]
#### Runtime Environment

- Operating system and version: Arch Linux (kernel 4.14.15-1-ARCH)
- Terminal emulator and version: termite v13
- Python version: 3.6.4
- Ranger version/commit: ranger version: ranger-stable 1.9.0
Python version: 3.6.4 (default, Jan  5 2018, 02:35:40) [GCC 7.2.1 20171224]
- Locale: en_GB.UTF-8

#### Current Behavior
Creating a new git repository with `git init` in a directory already containing files makes ranger crash, giving the error code in the Traceback below.

#### Expected Behavior
ranger should continue to work after the repo has been created.

#### Context
I'm not sure what to put here, as I believe I've described the issue sufficiently.

#### Steps to reproduce
1. create a bunch of files in a test directory
2. `git init` in the directory
3. ranger crashes

#### Traceback
```
ranger version: ranger-stable 1.9.0
Python version: 3.6.4 (default, Jan  5 2018, 02:35:40) [GCC 7.2.1 20171224]
Locale: en_GB.UTF-8
Current file: '/home/alex/gittutorial'

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.6/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.6/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.6/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.6/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.6/site-packages/ranger/gui/widgets/statusbar.py", line 104, in draw
    self._calc_bar()
  File "/usr/lib/python3.6/site-packages/ranger/gui/widgets/statusbar.py", line 109, in _calc_bar
    self._get_left_part(bar)
  File "/usr/lib/python3.6/site-packages/ranger/gui/widgets/statusbar.py", line 193, in _get_left_part
    directory.vcs.rootvcs.repotype, directory.vcs.rootvcs.branch)
TypeError: unsupported format string passed to NoneType.__format__

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by gaddox** [comment]
Works on 1.9.1, try updating to current `sudo pacman -S ranger`.

**Comment by Ablesius** [comment]
seems fixed in 1.9.1, thanks!

**Comment by toonn** [comment]
Not sure this actually got fixed, since @Ablesius doesn't have the problem anymore I'm closing the issue. Feel free to reopen if the problem resurfaces.
